libs/core/docs/reference/pins/spi-frequency.md

```diff
@@ -8,8 +8,18 @@ pins.spiFrequency(1000000);
 
 ## Parameters
 
-* **frequency**: a [number](/types/number) to set as the frequency for SPI bus clock. This value is the number of clock changes per second (Hz).
+* **frequency**: a [number](/types/number) to set as the frequency for the SPI bus clock. This value is the number of clock changes per second (Hz).
+
+## Example
+
+Set the SPI clock frequency to `1` megahertz.
+
+```blocks
+pins.spiFrequency(1000000);
+```
 
 ## See also
 
-[SPI](https://developer.mbed.org/handbook/SPI)
+[spi mode](/reference/pins/spi-mode)
+
+[SPI handbook](https://developer.mbed.org/handbook/SPI)
```

Comment:
We don’t use mbed anymore aside from Microbit. Remove

Reply:
ok...